ColorHighlighter.py

```diff
@@ -137,10 +137,40 @@ def region_name(s, is_text):
 
 def read_file(fl):
     try:
-        with codecs.open(fl, "r", "utf-8") as f:
+        with codecs.open(fl, "r", "utf_8") as f:
```

Comment:
I picked 'utf_8' as that's how it's listed in the Python documentation (https://docs.python.org/3/library/codecs.html), no other reason. I agree that utf-8 looks nice, so I'll switch it back :)